ocss.sh

```diff
@@ -45,15 +45,18 @@ upload_retries="1"
 # copy URL to clip board -- true/false
 copy_url="true"
 # command to open the uploaded screenshot in a browser
-open_command="chromium %url"
+open_command="open %url"
```

Comment:
This looks like it's the MacOS equivalent of `xdg-open`, which is a nice idea.  We would need some more changes elsewhere in the code to go down this route, eg. `if is_mac; then open $url; else xdg-open $url; fi`

For now, it's meant to be a user-configurable variable, and `open` doesn't exist on any other Unix.